src/puya/awst/nodes.py

```diff
@@ -2,7 +2,6 @@
 import enum
```

Comment:
Is it worth dropping the break and continue nodes to avoid ambiguity for future front ends?

Reply:
I think they're still valuable for straightforward loop construction, there's a bit of song and dance involved to replace their usage entirely with Goto.

To resolve the ambiguity, I've renamed the nodes now:
- `BreakStatement` -> `LoopExit`
- `ContinueStatement` -> `LoopContinue`
And also added a very short docstring noting that this applies to the innermost loop.